src/olympia/addons/api_urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import patterns, url
+
+from . import views
```

Comment:
Didn't we want to start putting API related views into `app.api`? I'd prefer that than mixing more "regular" (html) views with api-endpoints.